core/interop/pinvoke/marshalling/src/MarshallingSample/EnumMarshalling.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace MarshallingSample
+{
+    class EnumMarshalling
+    {
+        public static void Run()
+        {
+            Console.WriteLine("----- Enum marshalling samples -----");
+
+            // return 2
+            int count = MarshallingSampleNative.CountEnumFlags(MarshallingSampleNative.EnumFlags.A | MarshallingSampleNative.EnumFlags.C);
```

Comment:
This line is too long. The number of the characters is 126. Maybe we can introduce variable.

         var enumFlags =  MarshallingSampleNative.EnumFlags.A 
                          | MarshallingSampleNative.EnumFlags.C;
         int count = MarshallingSampleNative.CountEnumFlags(enumFlags);